src/client/pages/UserPage/components/UserProfile.tsx

```diff
@@ -41,6 +42,20 @@ class UserProfile extends React.Component<Props> {
     );
   };
 
+  pronounSelect: React.StatelessComponent<CustomFieldProps> = ({ input, className }) => {
+    const pronouns = [
```

Comment:
Lets re-use the values from `src/shared/UserEnums.ts` to avoid duplication errors.